test-utils/xhrMock.js

```diff
@@ -83,8 +82,8 @@ module.exports = function() {
 		$defineRoutes: function(rules) {
 			routes = rules
 		},
-		$defineJSONPCallbackKey: function(key) {
-			callback = key
+		$defineJSONPCallbackKey: function(/* key */) {
```

Comment:
It's commented out because the key wasn't used for anything. I feel it might be a programming error, though, thus it being commented out. (It's set in the parent scope to `"callback"`.)